src/offerings.ts

```diff
@@ -82,14 +82,17 @@ export interface PurchasesStoreProduct {
     readonly title: string;
     /**
      * Price of the product in the local currency.
+     * Contains the price value of defaultOption for Google Play.
      */
     readonly price: number;
```

Comment:
Should this be a `Price`?

Reply:
I believe it's a double: https://github.com/RevenueCat/purchases-hybrid-common/pull/327/files#diff-7a22824a34a6bb513678cf8fa4064903965a9fc50169fb3f51c47a5d9048344cR38. It's true however that in a `subscriptionOption` we have `pricingPhases` which do have a `Price`. These properties are mostly kept for compatibility purposes, specially with iOS.